tests/ProjectsMetadataTest.php

```diff
@@ -580,6 +573,74 @@ public function testProjectMemeberCannotManageUserAndSystemMetadata(string $role
         $this->assertCount(2, $this->client->listProjectMetadata($projectId));
     }
 
+    public function allowedListMetadataRoles(): array
```

Comment:
dej tomu prosím suffix `Provider` ať je to jasné

Reply:
Ako pride mi to trochu zbytocne, vzhladom k tomu ze tam su este dva a tie sa volaju podobne tka som to chcel spravit rovnako

Reply:
jde mi jen o naming convention. provider to je, tak by se tak mohl jmenovat. ale klidne to tak nech